File: 1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        int[][] ans = new int[m][n];
        for(int i=0;i<indices.length;i++){
            for(int j=0;j<m;j++){
                ans[j][indices[i][1]]++;
            }
            for(int j=0;j<n;j++){
                ans[indices[i][0]][j]++;
            }
        }
        int count =0;
        for(int i=0;i<m;i++){
            for(int j=0;j<n;j++){
                if(ans[i][j] %2 == 1){
                    count++;
                }
            }
        }
        return count;
    }
}

File: 1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.js
/**
 * @param {number} m
 * @param {number} n
 * @param {number[][]} indices
 * @return {number}
 */
var oddCells = function(m, n, indices) {
    let matrix = new Array(m).fill(0).map(() => new Array(n).fill(0));
    for(let i=0;i<indices.length;i++){
        for(let j=0;j<m;j++){
            matrix[j][indices[i][1]]++;
        }
        for(let j=0;j<n;j++){
            matrix[indices[i][0]][j]++;
        }
    }
    
    let count = 0;
    for(let i=0;i<m;i++){
        for(let j=0;j<n;j++){
            if(matrix[i][j] %2 !=0){
                count++;
            }
        }
    }
    return count;
};